inc/link.class.php

```diff
@@ -659,13 +669,45 @@ static function rawSearchOptionsToAdd($itemtype = null) {
 
       if (!Session::isCron()
           && !isCommandLine() && isset($_SESSION['glpiactiveentities_string'])) {
-         $newtab['joinparams']['condition'] = getEntitiesRestrictRequest('AND', 'NEWTABLE');
+         if ($itemtype == User::getType()) {
+            // User would need a different query as entities_id is set on the user's profiles
```

Comment:
If query is not valid, it should be fixed.

Reply:
No longer relevant after last commit.